_data/projects/Maven-Using-CMD.yml

```diff
@@ -0,0 +1,12 @@
+name: Maven-Using-CMD
+desc: Creating a Hello World code project in Java using the Apache Maven build tool and automating it using Jenkins.
+site: https://github.com/NishkarshRaj/Maven-Using-CMD
+tags:
+- Maven
+- Java
+- XML
+- Jenkins
+- Automation
+upforgrabs:
+  name: Help Needed 
```

Comment:
@NishkarshRaj Also, change this to `Up-for-Grabs`

Reply:
@ritwik12 Updated the label.
Please review.

Reply:
@NishkarshRaj Not that, change line 11